cmd/fyne/commands/get.go

```diff
@@ -26,58 +40,33 @@ func NewGetter() *Getter {
 	return &Getter{}
 }
 
-// Get automates the download and install of a named GUI app package.
-func (g *Getter) Get(pkg string) error {
-	return get(pkg, g.icon)
-}
-
-// AddFlags adds available flags to the current flags parser
-func (g *Getter) AddFlags() {
-}
-
-// PrintHelp prints help for this command when used in a command-line context
-func (g *Getter) PrintHelp(indent string) {
-	fmt.Println(indent, "The get command downloads and installs a Fyne application.")
-	fmt.Println(indent, "A single parameter is required to specify the Go package, as with \"go get\"")
-}
-
-// Run is the command-line version of the Get(pkg) command, the first of the passed arguments will be used
-// as the package to get.
-func (g *Getter) Run(args []string) {
-	if len(args) != 1 {
-		fmt.Fprintln(os.Stderr, "Missing \"package\" argument to define the application to get")
-		os.Exit(1)
-	}
-
-	err := get(args[0], g.icon)
-	if err != nil {
-		fmt.Fprintf(os.Stderr, "%s\n", err.Error())
-		os.Exit(1)
-	}
-}
-
 // SetIcon allows you to set the app icon path that will be used for the next Get operation.
 func (g *Getter) SetIcon(path string) {
 	g.icon = path
 }
 
-func get(pkg, icon string) error {
+// Get automates the download and install of a named GUI app package.
+func (g *Getter) Get(pkg string) error {
 	path := filepath.Join(goPath(), "src", pkg)
 
 	cmd := exec.Command("go", "get", "-u", "-d", pkg)
 	cmd.Env = append(os.Environ(), "GO111MODULE=off")
+	cmd.Stdout, cmd.Stderr = os.Stdout, os.Stderr
 
-	_, err := cmd.CombinedOutput()
+	err := cmd.Run()
+	if err != nil {
+		return err
+	}
 
 	if !util.Exists(path) { // the error above may be ignorable, unless the path was not found
 		return err
 	}
 
-	install := &installer{srcDir: path, icon: icon, release: true}
-	err = install.validate()
-	if err != nil {
+	install := &installer{srcDir: path, icon: g.icon, release: true}
+	if err := install.validate(); err != nil {
 		return errors.Wrap(err, "Failed to set up installer")
 	}
+
 	return install.install()
```

Comment:
If `Get` installs, why is `Install` a different command in content? Perhaps `Install` needs a new name?

Reply:
I think that's outside of the scope of this PR.